### ISSUE TITLE
First view of the application

### ISSUE THREAD

**Issue by choisar4** [HarryC493/CSC207Project#25]
I think it would make sense for the first view of the application to be the Sign In View. That way, users who have already registered can sign in without register again. We can then add a "Register Option" to this view for users who have not yet registered.

**Comment by HarryC493** [comment]
I agree. We can discuss next week about implementing this use case.

**Comment by lisazhong13** [comment]
Agree, that wouldn't be hard, we can just switch the view and add a new button
